includes/defaults.inc.php

```diff
@@ -844,6 +844,7 @@
 
 // General GUI options
 $config['gui']['network-map']['style']                  = 'new';//old is also valid
+$config['theme_icon_colour']                            = 'black'; // Font Awesome icon colour customisation
```

Comment:
This feels odd.  Shouldn't color be mostly dependent on the spot where they are used?
Also, shouldn't this be in CSS, no PHP?